### PR TITLE
Windows: don't fail the connection for recoverable errors on network changes

### PR DESCRIPTION
**Pull request by alalamav**
* Prevents the connection for failing during network changes when:
  1. A new network interface comes up, but the current (gateway) interface is still connected.
  2. We fail to add the route to tun2socks (router), since the route already exists.

### CODE REVIEW

File: tools/OutlineService/OutlineService/OutlineService.cs [hunk 1]
```diff
@@ -782,11 +798,10 @@ private void NetworkAddressChanged(object sender, EventArgs evt)
             {
                 AddIpv4Redirect(routerIp);
             }
-            catch (Exception)
+            catch (Exception e)
             {
-                eventLog.WriteEntry("Failed to configure IPv4 routes", EventLogEntryType.Error);
-                ResetRoutingOnNetworkError();
-                return;
+                // Do not fail the connection. The route most likely already exist.
```
Comment (trevj) [inline tools/OutlineService/OutlineService/OutlineService.cs]: typo: exists

Comment (alalamav) [reply tools/OutlineService/OutlineService/OutlineService.cs]: Done.

**Comment by alalamav** [comment]
Thanks for the review, @trevj. Since I only updated comments, should I recompile and sign `OutlineService.exe`?

**Comment by trevj** [comment]
No, no need to recompile since you just changed comments. Thanks!

**Pull request merged**